test/mlir/conversion/onnx_to_krnl/Sequence/Sequence_with_dealloc.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: onnx-mlir-opt -O3 --shape-inference --convert-onnx-to-krnl --buffer-deallocation-pipeline  %s -split-input-file | FileCheck %s
+// RUN: onnx-mlir-opt -O3 --shape-inference --convert-onnx-to-krnl --buffer-deallocation  %s -split-input-file | FileCheck %s
```

Comment:
I am not sure if this is related, but `buffer-deallocation-pipeline` is supposed to be used after `one-shot-bufferize`.